isotovideo

```diff
@@ -349,6 +349,19 @@ while ($loop) {
                     }})->{tags};
             next;
         }
+        if ($rsp->{cmd} eq 'verify_sound_image') {
```

Comment:
please note. Just prefix the query with backend_ and let isotovideo pass it as is